src/Raven.Studio/wwwroot/Content/css/pages/databases.less

```diff
@@ -330,3 +330,9 @@
 #restorePointSpinner {
     margin-left: 10px;
 }
+
+#compactDatabaseDialog {
+    .limit-height {
```

Comment:
I think we have already class for that: max-height so somethign like that ?

Reply:
There is an existing limit-height class with 250px 
```
.limit-height {
    max-height: 250px;
    overflow-y: auto;
}
```

Reply:
ok